### PR TITLE
Feature/form/select from api

### PR DESCRIPTION
**Pull request by faejr**
Allow fetching select options from the wordpress API

### CODE REVIEW

File: source/js/front/form.js [hunk 1]
```diff
@@ -1,91 +1,130 @@
...
+                    subFields[subFields.length - 1].remove();
+                }
+                if (subFields.length === 1) {
+                    removeButton.classList.add('u-display--none');
+                }
+            });
+        });
+    },
+    setupRemoteSelect: (form) => {
+        var apiUrl = eventintegration.apiurl;
```
Comment (nRamstedt) [inline source/js/front/form.js:104]: - 'var' instead of 'let' typo? 
- add check eg. 'eventintegration?.apiurl' and bailout if its undefined

**Review by nRamstedt — CHANGES_REQUESTED** [review]
works! (as usual) minor changes requested. rebase & merge : )

File: source/js/front/form.js [hunk 2]
```diff
@@ -1,91 +1,132 @@
...
+                    subFields[subFields.length - 1].remove();
+                }
+                if (subFields.length === 1) {
+                    removeButton.classList.add('u-display--none');
+                }
+            });
+        });
+    },
+    setupRemoteSelect: (form) => {
+        if (eventintegration?.apiurl === undefined) {
```
Comment (nRamstedt) [inline source/js/front/form.js:104]: hmm .. i'm not 100% sure if undefined will work? (maybe it does) but will look better with if(!eventintegration?.apiurl) imo. 

On second thoughts this function depends on an outer scope variable (eventintegration), perhaps its better to include it as an argument and also destruct it (eg. setupRemoteSelect: (form, { apiurl }) => () then we can just make the check if (!apiurl) return.

**Review by nRamstedt — CHANGES_REQUESTED** [review]

File: source/js/front/form.js (hunk 2 above)
Comment (faejr) [reply source/js/front/form.js:104]: destructing the argument does seem like the better solution here tbh, but we should just be depending on soemthing inside the library and it's a bit confusing

**Review by nRamstedt — APPROVED** [review]

**Pull request merged**